dev/Scroller/Scroller.cpp

```diff
@@ -696,23 +696,32 @@ winrt::Size Scroller::ArrangeOverride(winrt::Size const& finalSize)
             content.DesiredSize().Height
         };
 
+        bool wasContentArrangeWidthStretched = false;
```

Comment:
>bool wasContentArrangeWidthStretched = false; [](start = 8, length = 45)

Code like this is much more reviewable and readable if these boolean flags are declared as const. While reviewing this I find myself having to look though the whole change to see where the value might change. That isn't so bad for this review because it is relatively small, but for larger changes or if I had to go and make a change to this code in the future it would be much easier if I was confident that this value wasn't changing out from under me.

You could update this to be a const variable if you defined it via a lambda :)

Reply:
lambdas += 3;
consts += 3;
Thanks!